src/widgets/current-refined-values/__tests__/defaultTemplates-test.js

```diff
@@ -0,0 +1,44 @@
+import defaultTemplates from '../defaultTemplates.js';
+
+describe('current-refined-values defaultTemplates', () => {
+  describe('`item` template', () => {
+    it('has a `item` default template', () => {
+      const item = {
+        type: 'disjunction',
+        label: 'Brand',
+        operator: ':',
+        name: 'Samsung',
+        count: 4,
+        cssClasses: {
+          count: 'ais-current-refined-values--count',
+        },
+      };
+      expect(defaultTemplates.item(item)).toMatchSnapshot();
+    });
+    it('wraps query refinements with <q>', () => {
+      const item = {
+        type: 'query',
+        label: 'Query',
+        operator: ':',
+        name: 'Samsu',
+        cssClasses: {
+          count: 'ais-current-refined-values--count',
+        },
+      };
+      expect(defaultTemplates.item(item)).toMatchSnapshot();
+    });
+    it('does not show `count` when query refinement', () => {
+      const item = {
+        type: 'query',
+        label: 'Query',
+        operator: ':',
+        name: 'Samsu',
+        count: 22, // should be ignored
+        cssClasses: {
+          count: 'ais-current-refined-values--count',
+        },
+      };
+      expect(defaultTemplates.item(item)).toMatchSnapshot();
```

Comment:
We shouldn't rely exclusively on snapshots. I think this is something we should move away from. We completely forget what we're testing.

Reply:
I agree with you. I hate snapshots, too for the same reasons.
However:
- The snapshotted thing here is a string so there is no very easy way to test it (no enzyme)
- the name of the test `it('wraps query refinements with <q>') can address the concern you have about forgetting the purpose of test
What do you think @francoischalifour ?

Reply:
You can test that the expected value are inside inside the string + snapshots. The intent of the test will be a bit clearer.